src/api/Jalasoft.TeamUp.Projects.API/Startup.cs

```diff
@@ -19,10 +19,13 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.AddScoped<IHealthService, HealthService>();
             builder.Services.AddScoped<IHealthRepository, HealthRepository>();
             builder.Services.AddScoped<IProjectsService, ProjectsService>();
+            builder.Services.AddScoped<IInvitationsService, InvitationsService>();
 #if DEBUG
             builder.Services.AddScoped<IRepository<Project>, ProjectsInMemoryRepository>();
```

Comment:
please use here the newly created IProjectsRespositoty instead of IRepository<Project>, just like you do for invitations repos

Reply:
done